test/toc-test.js

```diff
@@ -73,6 +73,55 @@ describe('Toc', function() {
         var level = Toc.helpers.getTopLevel($scope);
         expect(level).to.eql(6);
       });
+
+      it("returns the first level if include all specified", function() {
+        var $scope = $(
+          '<div>' +
+            '<h1></h1>' +
+            '<h2></h2>' +
+            '<h3></h3>' +
+            '<h4></h4>' +
+            '<h5></h5>' +
+            '<h6></h6>' +
+            '<h6></h6>' +
+          '</div>'
+        );
+        var level = Toc.helpers.getTopLevel($scope, 1, true);
+        expect(level).to.eql(1);
+      });
```

Comment:
In this example, the table of contents would only show the `<h1>` and the `<h2>`...that's the desired behavior?